test-apps/expo-router/app/location.js

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react';
+import { View, Text, Button } from 'react-native';
+import * as Location from 'expo-location';
+
+export default function Another() {
+  const [location, setLocation] = useState(null);
+
+  async function readLocation() {
+    let { status } = await Location.requestForegroundPermissionsAsync();
+    console.log('Permission status', status);
```

Comment:
Remove.

Reply:
this is in the test app, I'd prefer to keep it there especially that we want to test logging too